yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -502,7 +547,7 @@ describe('AVM simulator: transpiled Noir contracts', () => {
       const context = initContext({ env: initExecutionEnvironment({ calldata }) });
       jest
         .spyOn(context.persistableState.hostStorage.contractsDb, 'getBytecode')
-        .mockReturnValueOnce(Promise.resolve(addBytecode));
+        .mockReturnValue(Promise.resolve(addBytecode));
```

Comment:
Is mockReturnValue preferred over mockReturnValueOnce?

Reply:
No! But since I'm now getting the bytecode first to decide if it's AVM or ACVM I needed to change it.